src/GitHub.Logging/Logging/ILoggerExtensions.cs

```diff
@@ -26,5 +27,21 @@ public static void Assert(this ILogger logger, bool condition, string messageTem
 #pragma warning restore Serilog004
             }
         }
+
+        /// <summary>
+        /// Log any exceptions when a task throws.
+        /// </summary>
+        /// <param name="task">The <see cref="Task"/> to log exceptions from.</param>
+        /// <param name="log">The <see cref="ILogger"/> to use.</param>
+        public static void LogAndForget(this Task task, ILogger log)
```

Comment:
Why add this new method rather than add logging to the existing `Forget` method in `TaskExtensions`? Also its name suggests that the main purpose of this method is to log, but that's not true.

Reply:
The idea was for it to be equivalent to `FileAndForget` which logs to Visual Studio's error log, but that takes an instance of one of our loggers instead.

Having the existing `Forget` method log somewhere as well would be a good idea, but the question is where since it won't have an instance of a logger.

Could we open an issue to do this?

Reply:
Ok, that makes sense. IMO this should probably just be called `Forget` as well though, so there would be two versions: one that takes a logger and one that doesn't.